UVA adaptive round 18

hokinus opened the pull request:
UVA adaptive round 18

LucieContamin commented:
Hi @hokinus ,

Thank you for the submission, I have some questions:
- It seems that the "inc death" and "inc hosp" don't have the same number of horizon projected. `inc hosp` is projected for 53 weeks and `inc death` for 55 weeks. 52 weeks is the expected horizon, would it be possible to verify it?
- Also, for the `"sample"` output type:
   - it seems that the column `output_type_id` is not set to `NA` (some rows have `-1`), would it be possible to verify that, please?
   - And for the sample ID information in the run grouping and stochastic run, if I understand correctly:
       -  100 trajectories was provided
       - And for the grouping information: there is no stochasticity, every models runs has a different run_grouping that can be group by age_group, location, horizon, scenario_id, target or by another level for example group by age group, horizon, scenario. Did I understand correctly?
 
Please let me know if any questions or issues
Best, Lucie

hokinus commented:
Lucie, 
I have updated the file, trimmed the projection horizon and fixed the output_type_id. 
Different run_groupings correspond to single set of parameters of our model, but for each location the calibration step is done independently and the selection of trajectories for submission is done separately (so for example set of run_groupings for VA may be different than set for MD). The age groups are disaggregated from a single run and deaths are result of hospitalization postprocessing. So single run_grouping (as in independent model run) corresponds to  `location, horizon, scenario_id`.

hokinus commented:
I might got confused by grouping by/pairing usage here, but your understanding is correct. 
The trajectories with the same run_grouping are using the same parameter set. The selection of the trajectories is done independently for each location and I do not enforce the selection of the trajectories with the same parameter set across locations, that is why some of the run_grouping ids are shared between some of the locations and some are not. 
If that is a problem, then I can:
1. reassign run_groupings so they are unique between locations 
2. or enforce selection of the trajectories to be the same across locations, so then trajectories will be "paired" across locations as well, that may slightly reduce the spread of the trajectories selected for single locations though

Let me know what is preferred.

hokinus commented:
Update to have unique run_groupings per location.

LucieContamin commented:
Hi @hokinus ,

Thank you for the update and the additional information, it seems to be all good now!

Best, Lucie
